src/components/apphost.js

```diff
@@ -426,33 +426,26 @@ define(["appSettings", "browser", "events", "htmlMediaHelper", "webSettings"], f
         }
     };
 
-    var doc = self.document;
     var isHidden = false;
 
-    if (doc) {
-        if (void 0 !== doc.visibilityState) {
+    if (self.document) {
+        if (self.document.visibilityState !== undefined) {
```

Comment:
```suggestion
        if (self.document.hidden !== undefined) {
```

Reply:
We're checking for API support here, not status, I believe?

Reply:
Alright, I stumbled on the source for that part of the code on MDN, it's a copy-paste from the "Example" section here: https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API

Reply:
I think we are just choosing proper field to check visibility state.